themes/vocabulary_theme/templates/blocks/hero.html

```diff
@@ -12,6 +12,6 @@ <h1>
     </div>
   </div>
   <div class="level-right hero-image">
-    <img class="image" src="./github.svg" />
+    <img class="image" src="./github.svg" alt="Hero image" />
```

Comment:
"hero image" is too generalized, if the image is purely decorative it shouldn't have alt text, if it is part of the content, then the contents of the hero image would need to be described appropriately.